Documentation/operations/upgrade.rst

```diff
@@ -402,6 +402,8 @@ The following metrics have been removed:
 * ``cilium_policy_count`` is removed. Please use ``cilium_policy`` instead.
 * ``cilium_policy_import_errors`` is removed. Please use ``cilium_policy_import_errors_total`` instead.
 * ``cilium_datapath_errors_total`` is removed. Please use ``cilium_datapth_conntrack_dump_resets_total`` instead.
+* ``cilium_operator_identity_gc_entries_total`` is removed. Please use ``cilium_operator_identity_gc_entries`` instead.
+* ``cilium_operator_identity_gc_runs_total`` is removed. Please use ``cilium_operator_identity_gc_runs`` instead.
```

Comment:
This is the section for the v1.10 upgrade guide. Please add this to the section for v1.11 upgrade guide (or create it if it doesn't exist)

Reply:
Update for v1.11 upgrade guide as well.

Reply:
As well? I don't think we plan to backport this to v1.10, please remove it from this section.

Reply:
Done.